### PR TITLE
chore(deps): bump penumbra to 0.80.5

### LINKED ISSUE #1390: update to penumbra tagged release when possible
we can update to a tagged penumbra release when https://github.com/penumbra-zone/penumbra/pull/4822 is included in a release.

┆Issue Number: ENG-737

### PR DESCRIPTION
**Pull request by noot**
## Summary
bump penumbra to latest release tag

## Related Issues

closes #1390

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1578,9 +1578,9 @@ dependencies = [
 
 [[package]]
 name = "celestia-tendermint"
-version = "0.32.2"
```
Comment (SuperFluffy) [inline Cargo.lock:1581]: This is going down for some reason - please fix?

File: Cargo.lock [hunk 2]
```diff
@@ -1607,9 +1607,9 @@ dependencies = [
 
 [[package]]
 name = "celestia-tendermint-proto"
-version = "0.32.2"
```
Comment (SuperFluffy) [inline Cargo.lock:1610]: Also this one is going down.

**Review by SuperFluffy — COMMENTED** [review]
Ready to merge the penumbra bits, but it's not clear why the celestia tendermint dependencies are having their versions decreased (neither sequencer nor penumbra have anything to do with those).

File: Cargo.lock (hunk 1 above)
Comment (noot) [reply Cargo.lock]: it was actually 0.32.1 in Cargo.toml - bumped the version there

Comment (SuperFluffy) [reply Cargo.lock]: This should not be necessary. Are you basing this patch on recent `main`? The entry in `Cargo.toml` indicates a minimum version. The entry in `Cargo.lock` indicates the version we actually use.

I have just forked off recent main and updated the penumbra entries - `celestia-tendermint` and `celestia-tendermint-proto` are not affected by the penumbra change.

If you take the current main, do the penumbra changes there, and then regenerate `Cargo.lock` you should get a clean patch.

**Review by SuperFluffy — APPROVED** [review]

File: Cargo.lock (hunk 1 above)
Comment (SuperFluffy) [reply Cargo.lock]: I rebased on top of main, reverted the last commit (for tendermint), and then restored && regenerated Cargo.lock. Now it does not contain tendermint changes.

**Comment by noot** [comment]
closing because of  #1691 for now - can bump penumbra once the height issue is figured out/upstreamed

**Pull request closed without merging**